code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -244,3 +242,17 @@
 	var/notice_delay = 20 //2 second between notices
 
 	var/fire_luminosity = 0 //Luminosity of the current fire while burning
+
+	//Last armor specific bonus(or debonus) applied on the mob
+	
+	//This tells what was some of the last applied armor bonus, uses it to remove before applying a new one, might as well turn into a list buuuuuuuuut lazy.
+	
+	//Warding Phero
+	var/lastwardingbonus = 0 
```

Comment:
no these are bad, these should be attached to the abilities not the xeno

Reply:
can't attach those to abilities since they aren't an ability, they're a pheromone that changes a value on the mob that then the mob adds armor based on it.